app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car_item: Car) -> float:
+        washing_price = car_item.comfort_class \
+            * (self.clean_power - car_item.clean_mark) \
+            * self.average_rating \
+            / self.distance_from_city_center
+        return round(washing_price, 1)
+
+    def wash_single_car(self, car_item: Car) -> None:
+        car_item.clean_mark = self.clean_power
+        pass
```

Comment:
please, remove `pass` statement from every method, it is unused (we add it to show you that you should implement these methods)